2024-05-11 14:46:10,818 - INFO - Test 'tests/test_FirstTaskPage.py::test_first_task_first' passed
2024-05-11 14:46:20,499 - ERROR - the link redirects to a third-party resource
2024-05-11 14:46:20,547 - ERROR - Test 'tests/test_FirstTaskPage.py::test_first_task_second' failed: browser = <selenium.webdriver.chrome.webdriver.WebDriver (session="5d8a61b861a265d59b5fda071c3f5a80")>
setup_logging = <Logger tests.conftest (DEBUG)>

    def test_first_task_second(browser, setup_logging):
        firsttaskpage = FirstTaskPage(browser)
        output = firsttaskpage.TextLinkOpen()
        # assert output[0] != None, setup_logging.error("link is invalid")
>       assert output[0] == output[1], setup_logging.error("the link redirects to a third-party resource")
E       AssertionError: None
E       assert None == 'https://tensor.ru/about'

tests\test_FirstTaskPage.py:13: AssertionError
